static/js/vue-cdr-access/src/components/browseSearch.vue

```diff
@@ -41,7 +41,12 @@
         methods: {
             getResults() {
                 let update_params = { anywhere: encodeURIComponent(this.search_query) };
-                this.$router.push({ name: 'displayRecords', query: this.urlParams(update_params) });
+                this.$router.push({ name: 'displayRecords', query: this.urlParams(update_params) })
+                    .catch((e) => {
+                        if (e.name !== 'NavigationDuplicated') {
```

Comment:
what triggers NavigationDuplicated errors? Something like clicking twice on a link?

Reply:
I think that or trying to redirect to the same url you're at. If I read the ticket mentioned in the commit correctly, it shouldn't give this error if you update parameters. It's possible that doesn't include parameters's specified via the ?. It might only refer to params in the route, like :uuid in our case.